pylib/cqlshlib/cqlshmain.py

```diff
@@ -2209,22 +2187,16 @@ def read_options(cmdlineargs, environment=os.environ):
         else:
             options.color = should_use_color()
 
-    if options.cqlversion is not None:
-        options.cqlversion, cqlvertup = full_cql_version(options.cqlversion)
-        if cqlvertup[0] < 3:
-            parser.error('%r is not a supported CQL version.' % options.cqlversion)
```

Comment:
Why is this removed?

Reply:
Passing anything other than the current CQL version (3.4.6 w/4.1) causes an error:

$ cqlsh --cqlversion 3.4.2
....
Connection error: ('Unable to connect to any servers', {'127.0.0.1:9042': ProtocolError("cql_version '3.4.2' is not supported by remote (w/ native protocol). Supported versions: ['3.4.6']")})